updateContact/index.ts

```diff
@@ -51,6 +51,7 @@ const httpTrigger: AzureFunction = async function (
 
       context.bindings.updateContact = contactDocument;
       context.bindings.contact = req.body;
+
```

Comment:
I'm not sure if this update method is doing anything at all. Where are you updating the existing contact?